Core/GUI/GameTooltipHooks.lua

```diff
@@ -515,14 +515,20 @@ local function onTooltipSetItem(tooltip, tooltipData)
 		return
 	end
 
-	local _, itemLink = tooltip:GetItem()
-	if type(itemLink) ~= "string" then
+	local name, link = tooltip:GetItem()
```

Comment:
Why rename the second return value? Can it also be some other type of link?

This page suggests that it can't: https://warcraft.wiki.gg/wiki/API_GameTooltip_GetItem